ci/Jenkinsfile.android

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-library 'status-jenkins-lib@v1.7.15'
+library 'status-jenkins-lib@switch_e2e_to_x86_64'
```

Comment:
@yakimant : should we get this merged in jenkins repo first?

Reply:
First, congrats with PR!

Regarding the jenkins lib - it switches the build from 32 to 64 bit.
Which is required to run on Android 12+ emulators.

As I understand you PR fixes issues on 64 bit emulators.
But is it compatible wit 32 bit still?

Another question is switching e2e emulators to Android 12+ 64bit.
Does it happen on SauceLab UI or in our repo?
@yevh-berdnyk, can you help?

Reply:
Hi,
No, it's not compatible with 32 anymore.
Regarding SayceLabs emulators - we're just changing Android version if we need it and that's it. But SauceLabs emulators are x86_64. E2e tests will still run after merging this PR

Reply:
hmm @yevh-berdnyk : thats because the CI for this PR points to this branch of `status-jenkins-lib`
I'm not sure that will be the case if we revert this change.

Also we can't or shouldn't merge this PR without this change first being merged in  `status-jenkins-lib`
probably @jakubgs or @yakimant , @apentori  can confirm...